org.lflang/src/lib/core/reactor_threaded.c

```diff
@@ -721,6 +721,17 @@ void request_stop() {
     lf_mutex_unlock(&mutex);
 }
 
+/**
+ * Return true of r1 has precedence over r2, false otherwise.
```

Comment:
```suggestion
 * Return true if the first reaction has precedence over the second, false otherwise.
 * @param r1 The first reaction.
 * @param r2 The second reaction.
```